phpunit.xml.dist

```diff
@@ -33,6 +33,7 @@ It's auto-generated by sonata-project/dev-kit package.
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
         <env name="SYMFONY_PHPUNIT_REMOVE" value="symfony/yaml" />
         <env name="SYMFONY_PHPUNIT_VERSION" value="8.5.0" />
+        <env name="KERNEL_CLASS" value="Sonata\Doctrine\Tests\Fixtures\App\Kernel" />
```

Comment:
Maybe we could keep this list in alphabetical order.

Reply:
This file should not be edited as mention in top of it. It's coming from https://github.com/sonata-project/dev-kit/blob/master/project/phpunit.xml.dist

Reply:
And how do I add this env var? Is there another way? Because symfony depends on this variable to load the kernel

Reply:
Solved with a workaround to not edit this file